Makefile

```diff
@@ -2,13 +2,29 @@ CF_FLAGS=--verbose
 DOCKER=docker
 BUILDFLAGS=
 RUNFLAGS=
-VERSION=3.6.5-1
-OWNER=bahamat
+VERSION=3.8.1-1
+OWNER=tzz
```

Comment:
I set the owner as a variable so that you could run

    make OWNER=myself

 to generate an image under your namespace. If you switch this back to me I'll merge.